go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/prometheus/common v0.9.1
 	github.com/rivo/uniseg v0.2.0 // indirect
 	github.com/shirou/gopsutil v3.21.2+incompatible
-	github.com/sirupsen/logrus v1.6.0
+	github.com/sirupsen/logrus v1.6.0 // indirect
```

Comment:
Can we remove it now?

Reply:
Indirect can not be removed. It is added by go tools.

Reply:
In my local environment, I can remove this line by manual, then run `make tidy` will delete the related line in `go.sum`.

Reply:
I'll keep `indirect` keyword here.

Reply:
per `go help modules`:
```
As part of maintaining the require statements in go.mod, the go command
tracks which ones provide packages imported directly by the current module
and which ones provide packages only used indirectly by other module
dependencies. Requirements needed only for indirect uses are marked with a
"// indirect" comment in the go.mod file. Indirect requirements are
automatically removed from the go.mod file once they are implied by other
direct requirements. Indirect requirements only arise when using modules
that fail to state some of their own dependencies or when explicitly
upgrading a module's dependencies ahead of its own stated requirements.
```

Reply:
Yeah I think it would be better to remove indirect dependencies.

Reply:
```
$ go mod why github.com/sirupsen/logrus
# github.com/sirupsen/logrus
github.com/pingcap/tidb/ddl/util
github.com/pingcap/tidb/ddl/util.test
go.etcd.io/etcd/integration
go.etcd.io/etcd/embed
github.com/tmc/grpc-websocket-proxy/wsproxy
github.com/sirupsen/logrus
```

`go mod tidy` will add it back due to etcd dependency maybe.